rules/rules.libsonnet

```diff
@@ -14,11 +14,11 @@
             // Reduces cardinality of this timeseries by #cores, which makes it
             // more useable in dashboards.  Also, allows us to do things like
             // quantile_over_time(...) which would otherwise not be possible.
-            record: 'namespace_pod_container:container_cpu_usage_seconds_total:sum_rate',
+            record: 'namespace_node_pod_container:container_cpu_usage_seconds_total:sum_rate',
             expr: |||
               sum by (namespace, pod, container) (
                 rate(container_cpu_usage_seconds_total{%(cadvisorSelector)s, image!="", container!="POD"}[5m])
-              )
+              ) * on (namespace, pod) group_left(node) kube_pod_info
```

Comment:
It can happen that during rollouts or other circumstances there might be two instances of kube-state-metrics temporarily, which would cause this rule evaluation to fail. Let's add a `max by(namespace, pod, node) (kube_pod_info)` on the right hand side.